deployment/kube/stage/frontend.yaml

```diff
@@ -43,6 +54,13 @@ spec:
         ports:
         - name: http-server
           containerPort: 9091
+        resources:
+         limits:
+           cpu: 600m
```

Comment:
Do we need to limit CPU usage? Why not let the pod use as much CPU as is available?